qiita_pet/templates/private_studies.html

```diff
@@ -134,6 +137,22 @@ <h4 class="modal-title" id="myModalLabel">Modify Sharing Settings</h4>
             {% end %}
         </tbody>
     </table>
+
+<!--User study Modals-->
+{% for s in user_studies %}
```

Comment:
Instead of adding all these hidden objects to the DOM, can we instead do this through JavaScript? Also, can you add a close button to the modal window?